bundles/org.openhab.binding.electroluxappliances/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,472 @@
+	</channel-type>
+
+	<channel-type id="filterLife">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Filter Life</label>
+		<description>The filter life indication.</description>
+		<state readOnly="true" min="0" max="100" pattern="%d %unit%"/>
+	</channel-type>
+
+	<channel-type id="doorOpen">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanSpeed">
+		<item-type>Number</item-type>
+		<label>Fan Speed Setting</label>
+		<description>The fan speed setting.</description>
+		<state>
+			<options>
+				<option value="1">Level 1</option>
+				<option value="2">Level 2</option>
+				<option value="3">Level 3</option>
+				<option value="4">Level 4</option>
+				<option value="5">Level 5</option>
+				<option value="6">Level 6</option>
+				<option value="7">Level 7</option>
+				<option value="8">Level 8</option>
+				<option value="9">Level 9</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="workMode">
+		<item-type>String</item-type>
+		<label>Work Mode Setting</label>
+		<description>The work mode setting.</description>
+		<state>
+			<options>
+				<option value="PowerOff">Power Off</option>
+				<option value="Auto">Automatic</option>
+				<option value="Manual">Manual</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="ionizer">
+		<item-type>Switch</item-type>
+		<label>Ionizer Status</label>
+		<description>The ionizer status.</description>
+	</channel-type>
+
+	<channel-type id="uiLight">
+		<item-type>Switch</item-type>
+		<label>UI Light</label>
+		<description>The air quality light status.</description>
+	</channel-type>
+
+	<channel-type id="safetyLock">
+		<item-type>Switch</item-type>
+		<label>Safety Lock</label>
+		<description>The safety lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorLock">
+		<item-type>Switch</item-type>
+		<label>Door Lock</label>
+		<description>The door lock status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="doorState">
+		<item-type>Contact</item-type>
+		<label>Door Status</label>
+		<description>The door status Open/Closed.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="connectionState">
+		<item-type>Switch</item-type>
+		<label>Connection State</label>
+		<description>The connection state status.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="startTime">
+		<item-type>Number:Time</item-type>
+		<label>Start Time</label>
+		<description>The start time of a washing.</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="timeToEnd">
+		<item-type>Number:Time</item-type>
+		<label>Time To End</label>
+		<description>The time to end of a program.</description>
+		<state readOnly="true"/>
```

Comment:
Please provide a state pattern and if not the default unit a unithint  (seconds is default)

Reply:
It is seconds.

Reply:
Fixed!

Reply:
Both time-to-start and time-to-end are missing `unitHint`

Reply:
Should I then add unitHint="s"?